### PR TITLE
USM Windows non-IIS services no longer in beta

### PR DESCRIPTION
**Pull request by shanelhuang**
feature is no longer in beta, and enabled by default on Windows non-IIS hosts

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/universal_service_monitoring/setup.md [hunk 1]
```diff
@@ -730,48 +727,36 @@ If you use load balancers with your services, enable additional cloud integratio
...
-
-The following systems or services require additional configuration:
-
-{{< collapse-content title="Non-containerized services on Linux" level="h4" >}}
-<div class="alert alert-info">
-Universal Service Monitoring is available in <strong>beta</strong> to monitor services running bare-metal on Linux virtual machines.
-</div>
+**For non-IIS services:**
 
 Requires Agent version 7.42 or greater.
```
Comment (guyarb) [inline content/en/universal_service_monitoring/setup.md:732]: we might want to highlight that versions prior to 7.57 but post 7.42 need the configuration enabled

Comment (shanelhuang) [reply content/en/universal_service_monitoring/setup.md]: thanks edited to mention this

File: content/en/universal_service_monitoring/setup.md [hunk 2]
```diff
@@ -730,48 +727,29 @@ If you use load balancers with your services, enable additional cloud integratio
...
-</div>
-
-Requires Agent version 7.42 or greater.
-
-{{< tabs >}}
-{{% tab "Configuration file" %}}
+**For non-IIS services:**
 
-Add the following configuration to the `system-probe.yaml`:
+Discovery of non-IIS services is enabled by default starting with Agent version 7.57. Previous agent versions may require the following configuration change to `system-probe.yaml`:
```
Comment (hestonhoffman) [inline content/en/universal_service_monitoring/setup.md:732]:
```suggestion
Discovery of non-IIS services is enabled by default starting with Agent version 7.57. Previous Agent versions may require the following configuration change to `system-probe.yaml`:
```

**Review by hestonhoffman — APPROVED** [review]
Tiny edit, but this LGTM!

**Pull request merged**